### ISSUE TITLE
More Evaluation Measures: Calibration score and D statistic

### ISSUE THREAD

**Issue by DL854359743057** [CamDavidsonPilon/lifelines#994]
Hi,
Thanks for this awesome package!
My Question:
How is it possible to calculate Calibration score and D statistic with lifelines?

**Comment by CamDavidsonPilon** [comment]
No, at least, not under those names. Can you provide definitions for those two metrics?

**Comment by DL854359743057** [comment]
In definitions are in this paper:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5395888/

1. Royston and Sauerbrei’s D
2. Calibration slope
